test/kino/control_test.exs

```diff
@@ -109,6 +109,19 @@ defmodule Kino.ControlTest do
       events = button |> Kino.Control.stream() |> Enum.to_list()
       assert events == [%{origin: "client1"}]
     end
+
+    test "supports Kino.JS.Live" do
+      kino = Kino.TestModules.LiveCounter.new(0)
+
+      spawn(fn ->
+        Process.sleep(1)
+        Kino.TestModules.LiveCounter.bump(kino, 1)
+        Kino.TestModules.LiveCounter.bump(kino, 2)
```

Comment:
It is best to write a busy loop that emits events every 10 miliseconds and then you assume two are received. Otherwise there is  no guarantee the stream will start before you emit them. Then you kill the process at the end of the test. :)

Reply:
Something also nice to test is that after we take 2, nothing leaks to the inbox, so we can do an `assert_receive`. Once we have a more robust test, we can ship it!

Reply:
Smart! I updated all similar tests with a background loop. For the leak I assume you meant `refute_receive`, or do you have an alternative in mind?

Reply:
refute_receive is excellent. You should prefer refute_received whenever possible (to avoid waiting) but in this case waiting a bit doesn't hurt (you can wait twice the time of the busy loop)